tests/embodied/kinematics/test_urdf_model.py

```diff
@@ -21,10 +21,13 @@ class VectorType(Enum):
     TH_VECTOR = 2
 
 
+device = "cuda:0" if torch.cuda.is_available() else "cpu"
```

Comment:
Since cuda is set explicitly should we also update the CI config to handle this just like test_theseus_layer.py? I am wondering if there is an easier way to set this up say with pytest.mark, than running each file individually from CI that have explicit cuda related tests. @luisenp

Reply:
Based on offline discussion, we can leave this as is. CI for these tests on cuda as well on top of current cpu tests would not be necessary.